package.json

```
@@ -25,8 +25,16 @@
         "url": "https://github.com/tonlabs/TON-Surf-Localization/issues"
     },
     "homepage": "https://github.com/tonlabs/TON-Surf-Localization#readme",
-    "dependencies": {
-        "@tonlabs/localization": "^2.3.0",
-        "typescript": "4.1.2"
+    "peerDependencies": {
+        "@react-native-clipboard/clipboard": "1.8.4",
+        "@tonlabs/uikit.localization": "2.3.2",
+        "react-native-localization": "2.1.7"
+    },
+    "dependencies": {},
+    "devDependencies": {
+        "@types/react": "17.0.24",
+        "@tonlabs/uikit.localization": "2.3.2",
```

Need to revert it back
```suggestion
        "@tonlabs/localization": "^2.3.2",
```